src/Auth/Contracts/Authenticator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth\Contracts;
+
+interface Authenticator
+{
+    public function login(Authenticable $authenticable): void;
+
+    public function logout(): void;
+
+    public function user(): ?Authenticable;
```

Comment:
This will need to be renamed depending on what we name `Authenticable`

Reply:
Got it!